test/specs/scale.linear.tests.js

```diff
@@ -788,7 +788,7 @@ describe('Linear Scale', function() {
 		chart.options.scales.y.max = 2.8;
 		chart.update();
 
-		expect(getLabels(chart.scales.y)).toEqual(['2.8', '2.5', '2.0', '1.5', '1.0', '0.5', '0.3']);
+		expect(getLabels(chart.scales.y)).toEqual(['2.8', '2.3', '1.8', '1.3', '0.8', '0.3']);
```

Comment:
I feel like some people might not be happy that we no longer get the "nice" ticks

Reply:
I think it would be expected to use the `stepSize` starting from defined `min`, rather than from zero? Can always specify a "nice" min to keep everything nice.

Reply:
One thing I wonder about is the zoom plugin though. There the user doesn't manually specify `min` and `max`, but the plugin specifies them after the user has used the mouse scroll wheel or dragged a box on the UI to zoom

Reply:
One way to do this might be to make the "nice" ticks into major ticks. Then the user could toggle between the two behaviors by turning major ticks on and off

Reply:
In the zoom plugin, step size isn’t set though